### ISSUE TITLE
Flashback CD - Crash

### ISSUE THREAD

**Issue by Hewhoisred** [MiSTer-devel/MegaCD_MiSTer#49]
The game crashes near the start of level 2.  You can get there yourself, enter "INKER" as the password.  During the first full motion video playback of the level, the game crashes with endlessly repeating audio and frozen video.

Here is a video of the bug:
https://youtu.be/enKgTpxMk3U

**Comment by marcopsf79** [comment]
I've detected the same problem but with random frozen FMV video (At some point, one FMV video will frozen). Manage to complete almost all missions but eventually it crashes.
It was my first MegaCD game on MiSTer....

**Comment by BrYaN55QG** [comment]
I have the same problem. The game freezes in the cinematic when I sit in the chair at the beginning of stage 2. It happens the same way as in the video Hewhoisred shared.

**Comment by CobolGuy007** [comment]
Recently tried Flashback on the SegaCD and i can confirm that the freezing still occurs, with the latest core version, when sitting in the chair, at the beginning of stage 2. It froze on me another time randomly during the intro as well on another FMV sequence.

**Comment by CobolGuy007** [comment]
Any news if this issue is being addressed? It still happens as per the latest core version, still not possible to bypass the crashing cinematic.

**Comment by birdybro** [comment]
The primary developer of this core is also simultaneously working on some improvements on the SNES core, finishing up the Saturn core, and an upcoming 3DO core. He is aware of the issues here though. If anyone else with the skills wants to attempt to tackle it they can submit a pull request.

**Comment by Nashismo** [comment]
Yes, I can confirm this issue, I was about to post it, but is already here... I have tried all the different CHD versions (floating on Archive.org) and even 1 to 1 bin/cue copies. But to no avail, the game crashes in the same spot, 2nd cinematic of level 2, when Conrad is sitting in the machine.
